io.sourceforge.clonekeenplus.yml

```
@@ -0,0 +1,36 @@
+app-id: io.sourceforge.clonekeenplus
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: CGeniusExe
+rename-desktop-file: cgenius.desktop
+rename-icon: cg
+
+modules:
+  - name: cgenius
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://gitlab.com/Dringgstein/Commander-Genius.git
+        tag: "v2.7.9"
+        commit: "b394882e791cb352976cb7b3deafc8f2dcb2e469"
+    config-opts:
+      - -DUSE_SDL2=yes
+      - -DUSE_BOOST=false
+      - -DSHAREDIR=/app/share
+      - -DAPPDIR=/app/bin
+    builddir: true
+    post-install:
+      - for s in 16 32 48 64 128 256 512; do install -D -m 644 -t $FLATPAK_DEST/share/hicolor/${s}x${s}/apps ../share/cg.${s}.png; done
+      - install -D -m 644 -t $FLATPAK_DEST/share/icons/hicolor/scalable/apps ../share/cg.svg
+      - install -D -m 644 -t $FLATPAK_DEST/share/metainfo ../share/metainfo/io.sourceforge.clonekeenplus.appdata.xml
+
+finish-args: 
+  - --device=all # needed for gamepads 
+  - --share=ipc	 # needed for X11
+  - --share=network # needed for networking (Gamestore, downloads)
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=home
```

```suggestion
  - --filesystem=home
```

Does it make sense to expose entire home if it uses just `~/.CommanderGenius`?

---

Yes, just changed that and did a local test.